### PR TITLE
Make sure variable bounds are json serializable

### PR DESCRIPTION
**Pull request by scottwittenburg**

### CODE REVIEW

**Comment by James-Crean** [comment]
LGTM. I think @doutriaux1's input will matter more than mine in this case though.

File: vcs_server/FileLoader.py [hunk 1]
```diff
@@ -70,12 +75,12 @@ def variables(self, file_name):
                     outVars[lonName] = {}
                 lonData = var.getLongitude()[:]
                 outVars[lonName]['bounds'] =\
-                  [numpy.amin(lonData), numpy.amax(lonData)]
+                  [n2p(numpy.amin(lonData)), n2p(numpy.amax(lonData))]
```
Comment (doutriaux1) [inline vcs_server/FileLoader.py]: why not simply use `float(min(lonData))`?

Comment (scottwittenburg) [reply vcs_server/FileLoader.py]: Yes that could possibly work and add less code.

**Comment by scottwittenburg** [comment]
@doutriaux1 To your comment about data possibly stored in reversed order: If we are taking the returned result in the first position to be the min, and the second position to be the max, those are still true and valid, are they not?  I guess it may depend on how client code uses the results.  Maybe we should be using a dictionary there so it's clear:

``` javascript
bounds = {
  min: float(min(lonData)),
  max: float(max(data)),
},
```

**Comment by scottwittenburg** [comment]
@doutriaux1 I guess this could be ready now.

**Pull request merged**